tests/ApplicationDefaultCredentialsTest.php

```diff
@@ -590,6 +590,58 @@ public function testGetCredentialsUtilizesQuotaProjectInKeyFile()
         );
     }
 
+    /** @runInSeparateProcess */
+    public function testGetCredentialsUtilizesQuotaProjectEnvVar()
+    {
+        $quotaProject = 'quota-project-from-env-var';
+        putenv(ServiceAccountCredentials::QUOTA_PROJECT_ENV_VAR . '=' . $quotaProject);
```

Comment:
I am new to PHP, so could be a dumb qn. Why is the QUOTA_PROJECT_ENV_VAR constant used from ServiceAccountCredentials and not from CredentialsLoader?

Reply:
No reason, only because it's used like that elsewhere in the tests. I agree that from CredentialsLoader makes more sense

Reply:
changed!